shell/common/switches.h

```diff
@@ -82,6 +82,9 @@ DEF_SWITCH(UseTestFonts,
            "will make font resolution default to the Ahem test font on all "
            "platforms (See https://www.w3.org/Style/CSS/Test/Fonts/Ahem/). "
            "This option is only available on the desktop test shells.")
+DEF_SWITCH(RunForever,
+           "run-forever",
+           "Keep the shell running after the Dart script has completed.")
```

Comment:
We should say that this is the default for interactive but not the default for `--non-interactive` mode.